backend/src/main/java/com/example/dontwastetomuch/game/GameController.java

```diff
@@ -21,9 +21,19 @@ public class GameController {
     public void addAGame(@RequestBody Game game){
         gameService.addAGame(game);
     }
-
+    
     @GetMapping
     public List<Game> getAllGames(){
         return gameService.getAllGames();
     }
+
+    @GetMapping("/{gameId}")
+    public Game getOneGame(@PathVariable String gameId){
+        return gameService.getOneGame(gameId);
+    }
```

Comment:
What will happen if no game is found? How your gonna tell your Frontend how to act?